lib/utils.ts

```diff
@@ -139,7 +147,7 @@ export function removeSlash(str: string | number): string {
 /**
  * Check if an object has a property
  */
-export function hasProperty(obj: any, prop: string): boolean {
+export function hasProperty(obj, prop: string): boolean {
```

Comment:
An explicit any is better than no type. In this case you should limit the argument to non-nullable values, because hasOwnProperty will throw otherwise:
```suggestion
export function hasProperty(obj: {}, prop: string): boolean {
```

Reply:
FYI, `{}` is NOT the empty object type, this is a bit confusing.
`{}` is basically `any` minus `undefined | null` and without disabling type checking.

Reply:
```
Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `Record<string, unknown>` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead.eslint@typescript-eslint/ban-types
```

When using `{}` as a type I get this

Reply:
> `{}` actually means "any non-nullish value"

Hm, you might actually want to limit it to objects, so `Record<string, unknown>` would be better.

Reply:
Also using `Record<string, unknown>` it's not working 🤷🏼‍♂️

Reply:
Disabled the rule for that line for now